boards/nxp/rd_rw612_bga/rd_rw612_bga.dtsi

```diff
@@ -91,3 +91,7 @@
 		};
 	};
 };
+
```

Comment:
Do we need to update the documentation to indicate this support

Reply:
Thanks for the reminder- board doc page updated